e2e/integration/spec/gateway/v2.spec.js

```diff
@@ -9,14 +9,14 @@ describe('Gateway v2 API', () => {
   const key = 'integration_tests/some_key';
 
   it('should get key using v2 (proxy to api service)', async () =>
-    await client.get(`/api/v2/values/${key}`).expect(200, '1.0'));
+    await client.get(`/api/v2/values/${key}`).expect(200, '1'));
```

Comment:
If it's possible to do the JSON parsing it might be more stable.
Check supertest (https://github.com/visionmedia/supertest) api